DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: miaViz
 Title: Microbiome Analysis Plotting and Visualization
-Version: 1.13.9
+Version: 1.14.0
```

Comment:
Version should be 1.13.10

1.xx.yy

- xx denotes the Bioc release/devel version (updates every 6 months)
- yy rolling number

I will check this PR tomorrow at latest